openapi-python-templates/model.mustache

```diff
@@ -1,5 +1,6 @@
 from datetime import datetime
 from typing import Optional, List
+from typing import Any  # noqa
```

Comment:
@Toumetis Is there a reason you needed to `# noqa` this? I don't think it should get removed by autoflake unless it is actually unused.

Maybe it was because it only occurs inside strings (and so wasn't being seen as used)?

Reply:
It's unused in the generated file, but the apis end up referring to 'm.Any' instead of 'Any' for some reason & the quick fix was to re-export Any from the model file.

Reply:
Aha; might make sense to just add a regex substitution downstream for `\bm.Any\b` to `Any` or similar. Happy to keep it like this for now though.